### PR TITLE
Missing work types

### PR DESCRIPTION
**Pull request by prdanelli**

### CODE REVIEW

File: app/forms/concerns/hyku_addons/work_form.rb [hunk 1]
```diff
@@ -17,15 +17,36 @@ def editor_list
     end
 
     class_methods do
```
Comment (okeen) [inline app/forms/concerns/hyku_addons/work_form.rb:19]: This looks like a really nice clean up. The only downside is it is going to whitelist every attribute for all the work types, right?

**Review by okeen — APPROVED** [review]
Looks great :)

File: app/forms/concerns/hyku_addons/work_form.rb (hunk 1 above)
Comment (prdanelli) [reply app/forms/concerns/hyku_addons/work_form.rb:19]: Yea, but i'm not sure thats an issue personally. It's better than missing some and then losing data IMO.

**Comment by prdanelli** [comment]
@cjcolvar Feel free to merge this monstrosity of a PR. The form fields and model properties are all correct. Presenters will need checking as i've been adding the form fields, which might not always be required.

**Pull request merged**